packages/met_office_site_archive/src/site_archive_met_office/__init__.py

```diff
@@ -0,0 +1,57 @@
+# Copyright Commonwealth of Australia, Bureau of Meteorology 2024.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+Met Office specific Indexes
+
+| Name        | Description |
+| :---        |       ----: |
+| [ERA5][site_archive_met_office.ERA5lowres]   | ECWMF ReAnalysis v5 WeatherBench Low Res       |
+| [MOGLOBAL][site_archive_met_office.MOGLOBAL] | Subset of Met Office Global Analysis Data      |
+| [MOUKV][site_archive_met_office.MOUKV]       | Subset of Met Office UKV Analysis Data         |
+"""
+
+import os
+import pyearthtools.data
+from pyearthtools.data.archive import register_archive, set_root_directory, get_root_directories, load_root_directories_from_config
+
+
+ROOT_DIRECTORIES = {
+    "ERA5lowres": "",
+    "MOGLOBAL": "", 
+    "MOUKV": "",        
+}
+
+register_archive("ROOT_DIRECTORIES")(ROOT_DIRECTORIES)
+
+import site_archive_met_office
+
+from site_archive_met_office.ERA5lowres import ERA5lowres
+from site_archive_met_office.MOGLOBAL import MOGLOBAL
+from site_archive_met_office.MOUKV import MOUKV
+
+register_archive("met_office")(site_archive_met_office)
+
+# Load the Met Office PyEarthTools config file to set the root directories
+username = os.getenv("USER")
+load_root_directories_from_config(config_path=f"/home/users/{username}/.pyearthtoolsconfig")
```

Comment:
```suggestion
config_path = os.path.expanduser("~/.pyearthtoolsconfig")
load_root_directories_from_config(config_path=config_path)
```
This avoids the need to handle usernames and environment variables and should be cross platform compatible.